src/Signature.xml

```diff
@@ -28,78 +26,75 @@
                 <translation lang="en_US">Reset signature</translation>
             </translations>
         </property>
-
-        <property key="penColor" type="string" defaultValue="#0000ff">
+        <property key="penColor" type="string" defaultValue="#000">
             <caption>Pen color</caption>
-            <category>Customization</category>
+            <category>Pen Customization</category>
```

Comment:
Have a tab for Datasource.. Appearance
Any thing to do with style should go to appearance